lib/zendesk2/help_center/article.rb

```diff
@@ -45,6 +45,10 @@ class Zendesk2::HelpCenter::Article
   attribute :vote_count, type: :integer # ro:yes required:no
   # @return [Integer] The total sum of votes on this article
   attribute :vote_sum, type: :integer # ro:yes required:no
+  # @return [Integer] The id of the user segment which defines who can see this article
+  attribute :user_segment_id, type: :integer, default: nil # ro:no required:yes
```

Comment:
According to the documentation, user_segment_id is required but can be null.
Therefore, set the default value and it is not included in the required items for save!.

![image](https://user-images.githubusercontent.com/12995110/61097187-3fdc5f00-a495-11e9-9767-88343a1f65e4.png)